test/algorithms/disjoint/disjoint_point_box_geometry.cpp

```diff
@@ -95,6 +94,29 @@ void test_all()
         "BOX(1941 2066, 2055 2166)", false);
 }
 
+template <typename P>
+void test_open()
+{
+    using box = bg::model::box<P>;
+    using ring_cw = bg::model::ring<P, true, false>;
+    //using polygon_cw = bg::model::polygon<P, true, false>;
+    //using mpolygon_cw = bg::model::multi_polygon<polygon_cw>;
+    //using ring_ccw = bg::model::ring<P, false, false>;
+    using polygon_ccw = bg::model::polygon<P, false, false>;
+    using mpolygon_ccw = bg::model::multi_polygon<polygon_ccw>;
+    
+    // https://github.com/boostorg/geometry/issues/837
+    test_disjoint<mpolygon_ccw, box>("issue_837",
+                                     "MULTIPOLYGON(((1992 3240,1992 1440,3792 1800,3792 3240,1992 3240)))",
+                                     "BOX(1941 2066, 2055 2166)", false);
+
+
+    // https://github.com/boostorg/geometry/issues/982
```

Comment:
Personally I don't use the URL's. An `issue_xxx` is always a github "issue". For the old svn trac system I used `ticket_xxx` because they are called "tickets" there.

https://github.com/boostorg/geometry/blob/develop/test/algorithms/overlay/multi_overlay_cases.hpp#L1500

Reply:
Ok, this information is indeed redundant.